src/sa_web/jstemplates/place-detail-story-bar-tagline.html

```diff
@@ -2,9 +2,9 @@
 	<section class="place-story-bar">
 		<table>
 			<tr>
-				<td class="btn-previous-story-nav {{#unless story.previous}}story-nav-disabled{{/unless}}"></td>
+				<td valign="top" class="btn-previous-story-nav {{#unless story.previous}}story-nav-disabled{{/unless}}"></td>
```

Comment:
@goldpbear  I think we should use the [vertical-align](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align). It looks like the HTML `valign` property is deprecated, as described here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td?redirectlocale=en-US&redirectslug=HTML%2FElement%2Ftd

And it makes sense, in that we should decouple our HTML content from our CSS styles. Let me know if you have any thoughts on this!